Aztec/Classes/NSAttributedString/Conversions/AttributedStringSerializer.swift

```diff
@@ -108,9 +108,25 @@ class AttributedStringSerializer {
         if let representation = implicitRepresentation(for: element, inheriting: childAttributes) {
             content.append(representation)
         } else {
-            for child in element.children {
-                let childContent = serialize(child, inheriting: childAttributes)
-                content.append(childContent)
+            // if the element is of type '.a' and contains a '.img'
+            if element.isNodeType(.a) && element.children.count == 1 {
+                if let imgElement = element.children.first as? ElementNode, imgElement.isNodeType(.img) {
+
+                    // get the link URL and assign it to the image attachment
+                    if let linkText = element.stringValueForAttribute(named: "href") {
+                        let imgAttributes = self.attributes(for: imgElement, inheriting: attributes)
+                        if let attachment = imgAttributes[NSAttachmentAttributeName] as? ImageAttachment,
```

Comment:
Logic: There's a problem here since if this is false, we're losing content.

The truth is that the code we currently have doesn't support this scenario very well, so I'd recommend to deal with the other comments, and once that's done let's discuss this part of the code in Slack to try and provide a nice solution for it.